code/modules/antagonists/traitor/datum_traitor.dm

```diff
@@ -251,6 +251,8 @@
 				traitor_won = FALSE
 			count++
 
+	result += "<br>Faction: <B>[employer]</B>"
```

Comment:
Can we have this more natural than "Key: Value", which always reads kinda lame?

Like, "Traitor from [employer]"

Reply:
I really want to but some of the employers are... not actually employers? It would come out as "Traitor from Gone Postal"

Reply:
What if the format of this was in the faction itself, actually.

Like you could specify in the faction for "Gone Postal" that it should be displayed as ", who went Postal"

Reply:
yeah sure